scripts/update_and_rebuild_libmesh.sh

```diff
@@ -28,6 +28,21 @@ if [[ $? == 0 && "x$git_dir" == "x" ]]; then
   fi
 fi
 
+# check if the user has ccache configured
+DISABLE_TIMESTAMPS=""
+echo $CXX | cut -d ' ' -f1 | grep '^ccache$' > /dev/null
+if [ $? == 0 ]; then
+  echo -n "ccache detected - "
+  # check if timestamps are explicitly enabled
+  echo "$* " | grep -- '--enable-timestamps ' > /dev/null
+  if [ $? == 0 ]; then
+    echo "warning: setting --enable-timestamps explicitly will negatively impact the ccache performance"
+  else
+    echo "configuring limbesh with --disable-timestamps to improve cache hit rate"
+    DISABLE_TIMESTAMPS="--disable-timestamps"
+  fi
```

Comment:
> What happens if both disable and enable are specified at the same time?

The latter takes precedence.

Actually, I think I agree with you now that I thought about it again.  Since `--enable-timestamps` is the default in libmesh, it's very unlikely that someone would explicitly specify it as a configure option.  So if they do specify it, it makes sense to assume that they *really, really* want it and respect that.